docs/developer/decision-records/2022-07-29-self-description/README.md

```diff
@@ -0,0 +1,59 @@
+# Identity Hub - Self Description
+
+This document explains how the Self-Description of a dataspace participant will be stored into the Identity Hub and retrieved from it.
+
+## Context
+
+Gaia-X defines a set of rules that define the minimum baseline to be part of a Gaia-X Ecosystem. This set of rules is the so-called
+[Gaia-X Trust Framework](https://gaia-x.gitlab.io/policy-rules-committee/trust-framework/), which is centered around the _Self-Description_ document.
+
+Each dataspace participant (consumer, provider, federator) is represented by a Self-Description document.
+
+According to [spec](https://gaia-x.eu/wp-content/uploads/2022/06/Gaia-x-Architecture-Document-22.04-Release.pdf),
```

Comment:
Should we document that we will initially serve presentations in JWT, rather than JSON-LD format?